src/ride/ride.c

```diff
@@ -1547,7 +1547,7 @@ void ride_construction_set_default_next_piece()
 			bank = FlatRideTrackDefinitions[trackType].bank_end;
 			slope = FlatRideTrackDefinitions[trackType].vangle_end;
 		} else {
-			curve = gTrackCurveChain[trackType].next;
+			curve = (ride->type!=RIDE_TYPE_WILD_MOUSE&&trackType==TRACK_ELEM_BOOSTER)?356:gTrackCurveChain[trackType].next;
```

Comment:
Lines like these get pretty hard to read on GitHub:
![image](https://cloud.githubusercontent.com/assets/737603/21638476/cefc7d22-d26b-11e6-86d0-bb6850d1a982.png)


Also, `356` is a magic number. How about something like this?
```
if (ride->type != RIDE_TYPE_WILD_MOUSE && trackType == TRACK_ELEM_BOOSTER) {
   curve = 0x100 | TRACK_ELEM_BOOSTER;
} else {
    curve = gTrackCurveChain[trackType].next;
}
```